### PR TITLE
Update dependencies

### PR DESCRIPTION
**Pull request by bandersen23**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [ ] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [ ] Ensured the license file is being packaged.

Adding additional dependencies to make sure package is close to `pypi` package

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -25,8 +25,9 @@ requirements:
     - python >=3.6
     - setuptools
     - flask >=1.0.4
-    - flask-compress
     - plotly >=5.0.0
+    - importlib-metadata ==4.8.3
+    - contextvars ==2.4
```
Comment (moorepants) [inline recipe/meta.yaml:30]: Hard pins are not going to work well in conda environments. I don't' think we should do this.

Comment (moorepants) [reply recipe/meta.yaml:30]: This https://github.com/plotly/dash/blob/v2.9.3/requires-install.txt shows:

```
Flask>=1.0.4
plotly>=5.0.0
dash_html_components==2.0.0
dash_core_components==2.0.0
dash_table==5.0.0
importlib-metadata==4.8.3;python_version<"3.7"
contextvars==2.4;python_version<"3.7"
```

Which seems to indicate that these two packages are only needed for Python < 3.7. Conda forge doesn't build for ython < 3.6, so we shouldn't need them.

Comment (moorepants) [reply recipe/meta.yaml:30]: This is the CI error:

```
The reported errors are:
- Encountered problems while solving:
-   - package contextvars-2.4-py_0 requires python >=3,<3.7, but none of the providers can be installed
```

**Comment by bandersen23** [comment]
@moorepants I removed the hard pins, let me know if this is fine. Thanks!

**Comment by moorepants** [comment]
Removing flask compress seems fine. LGTM

**Pull request merged**